File: src/main/java/com/chamodshehanka/springsample/topic/Topic.java
package com.chamodshehanka.springsample.topic;

/**
 * @author chamodshehanka on 3/6/2019
 * @project spring-sample
 **/
public class Topic {

    private String topicID;
    private String topicName;
    private String topicDescription;

    public Topic() {
    }

    public Topic(String topicID, String topicName, String topicDescription) {
        this.topicID = topicID;
        this.topicName = topicName;
        this.topicDescription = topicDescription;
    }

    public String getTopicID() {
        return topicID;
    }

    public void setTopicID(String topicID) {
        this.topicID = topicID;
    }

    public String getTopicName() {
        return topicName;
    }

    public void setTopicName(String topicName) {
        this.topicName = topicName;
    }

    public String getTopicDescription() {
        return topicDescription;
    }

    public void setTopicDescription(String topicDescription) {
        this.topicDescription = topicDescription;
    }
}

File: src/main/java/com/chamodshehanka/springsample/topic/Topic.kt
package com.chamodshehanka.springsample.topic

/**
 * @author chamodshehanka on 3/6/2019
 * @project spring-sample
 */
class Topic {

    var topicID: String? = null
    var topicName: String? = null
    var topicDescription: String? = null

    constructor() {}

    constructor(topicID: String, topicName: String, topicDescription: String) {
        this.topicID = topicID
        this.topicName = topicName
        this.topicDescription = topicDescription
    }
}
